packages/schematics/angular/application/files/src/polyfills.ts

```diff
@@ -40,6 +40,25 @@
 /** IE10 and IE11 requires the following for the Reflect API. */
 // import 'core-js/es6/reflect';
 
+/** IE9 and IE10 requires the following polyfill for using rxjs 6.0 */
```

Comment:
This polyfill is already included when you include the `IE9`, `IE10` and `IE11` polyfills above.

https://github.com/everedifice/angular-cli/blob/master/packages/schematics/angular/application/files/src/polyfills.ts#L23